wrappers/ios/Sources/Wrapper/Reader/ZXIResult.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 KURZ Digital Solutions GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#import <Foundation/Foundation.h>
+#import "ZXIFormat.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ZXIResult : NSObject
```

Comment:
would you be willing to add a few more useful properties from `Result`? Not necessarily all but especially the `position()` seems useful to me to help the user understand which barcode has just been found. There might be more than one which would lead to confusion.